mk/docs.mk

```diff
@@ -56,7 +56,7 @@ docs/generated/openapi.yaml:
 	rm -rf $(OAPI_TMP_DIR)
 	mkdir -p $(dir $@)
 	mkdir -p $(OAPI_TMP_DIR)/policies
-	for i in $(API_DIRS); do mkdir -p $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); cp -r $$(echo $${i} | cut -d: -f1) $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); done
+	for i in $(API_DIRS); do mkdir -p $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); cp -R $$(echo $${i} | cut -d: -f1) $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); done
```

Comment:
Is this supposed to be here? Please at least a different commit, ideally PR

Reply:
Yeah it is I couldn't generate the change openapi without this